build.sh

```diff
@@ -19,7 +19,7 @@ join -t , -1 2 -2 2 -o 1.1,0,2.3,2.4 <(sort -k 2 -t , target/pub_postcodes.csv)
 awk -F, -f filter_postcodes.awk target/named_pubs.csv | awk -F, -f split_pubs.awk
 
 #Add additional bits to "outcodes" for non postcode places
-awk 'BEGIN { FS="\t"; OFS=","; } $8 ~ /^(PPL|ADM)/ && $5 > 49 && $5 < 61 && $6 > -8 && $6 < 2 { gsub(/[^a-zA-Z0-9]/, "", $2); print $8, $2, $5, $6, $15 }' GB.txt \
+awk 'BEGIN { FS="\t"; OFS=","; } $8 ~ /^(PPL|ADM)/ && $5 > 49 && $5 < 61 && $6 > -8 && $6 < 2 { gsub(/[^a-zA-Z0-9]/, "", $2); printf "%s,%s,%.4f,%.4f,%s\n", $8, $2, $5, $6, $15 }' GB.txt \
```

Comment:
OBSERVATION: Great idea! I'd consider putting it into a `reduce_precision.awk` - so if we need to tweak it in future it can be changed in one place.

Reply:
I did consider that. But the change from this morning means they are different sized (4 Vs 5) so would be another call to awk after the cut happens on the bigger command or add it to the awk command which splits files. Not sure what's best here.

Reply:
I think a `reduce_precision.awk` would fit in nicely after the cut.